core_impl/src/main/java/io/opencensus/implcore/stats/MutableViewData.java

```diff
@@ -221,6 +225,16 @@ Timestamp getStart() {
   }
 
   /**
+   * Returns the milliseconds representation of this {@code Duration}.
+   *
+   * @param duration {@code Duration}.
+   * @return the milliseconds representation of a {@code Duration}.
+   */
+  static long toMillis(Duration duration) {
```

Comment:
There will be other classes using this method, so I think it's not strictly `VisibleForTesting`. I agree since this method is package-private, using javadoc instead of plain comments might be unnecessary. I've updated the comments here.